### PR TITLE
[Docs] Image Cog Guide

### PR DESCRIPTION
**Pull request by Kreusada**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature
- [x] Documentation 

### Description of the changes

This PR covers the cog guide for the Image cog - feel free to recommend any necessary changes.

(#1734 for tracker)

### CODE REVIEW

File: docs/cog_guides/image.rst [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+^^^
+gif
+^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]gif [keywords...]
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
    [p]gif <keywords...>
```

File: docs/cog_guides/image.rst [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+**Description**
+
+Retrieve the first search result from Giphy. This command requires API tokens
+to be set via the :ref:`giphycreds <image-command-giphycreds>` command.
+
+**Arguments**
+
+* ``[keywords...]``: The keywords used to search Giphy.
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
* ``<keywords...>``: The keywords used to search Giphy.
```

File: docs/cog_guides/image.rst [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+^^^^
+gifr
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]gifr [keywords...]
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
    [p]gifr <keywords...>
```

File: docs/cog_guides/image.rst [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+**Description**
+
+Retrieve a random GIF from a Giphy search. This command requires API tokens
+to be set via the :ref:`giphycreds <image-command-giphycreds>` command.
+
+**Arguments**
+
+* ``[keywords...]``: The keywords used to generate a random GIF.
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
* ``<keywords...>``: The keywords used to generate a random GIF.
```

File: docs/cog_guides/image.rst [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+""""""""""""
+imgur search
+""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]imgur search [count=1] [terms...]
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
    [p]imgur search [count=1] <terms...>
```

File: docs/cog_guides/image.rst [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+**Description**
+
+Search for pictures on Imgur. This command requires API tokens to be set
+via the :ref:`imgurcreds <image-command-imgurcreds>` command.
+
+**Arguments**
+
+* ``[count]``: How many images should be returned (maximum 5). Defaults to 1.
+
+* ``[terms...]``: The terms used to search Imgur.
```
Comment (Flame442) [inline docs/cog_guides/image.rst]:
```suggestion
* ``<terms...>``: The terms used to search Imgur.
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
The associated signatures and a few docstrings need to be updated for this cog, but that is not necessary for this PR. Just fix the bracket usage in this doc, and it's all set.

**Comment by Kreusada** [comment]
Thanks for reviewing, the requested changes have been made

**Review by Flame442 — APPROVED** [review]
Thanks for making those changes. This is all set!

**Pull request merged**